net-dns/https_dns_proxy/https_dns_proxy-0_pre20200925.ebuild

```diff
@@ -20,12 +20,23 @@ HOMEPAGE="https://github.com/aarond10/https_dns_proxy"
 LICENSE="MIT"
 SLOT="0"
 
-DEPEND="dev-libs/libev
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+COMMON_DEPEND="dev-libs/libev
 	net-dns/c-ares
 	net-misc/curl[http2,ssl]"
-RDEPEND="${DEPEND}"
+DEPEND="${COMMON_DEPEND}
+	test? ( dev-cpp/gtest )"
```

Comment:
by the way, tests blow up for me, very likely due to the presence of clang-tidy. Install clang and check for yourself, I recommend disabling the clang-tidy checks and adding an otpion with upstream.

Reply:
I've disabled clang-tidy check